InventoryImportExport/etc/di.xml

```diff
@@ -30,6 +30,9 @@
     <!-- Source Item Import Plugin -->
     <type name="Magento\CatalogImportExport\Model\StockItemImporterInterface">
         <plugin name="importStockItemDataForSourceItem" type="Magento\InventoryImportExport\Plugin\Import\SourceItemImporter" />
```

Comment:
Please check the original PR https://github.com/magento-l3/inventory/pull/106/files#diff-f270f253001fcad91997bf0df71fb7e3d1cba203c9e924459fd68684c7f7cb45L31. This plugin has been removed